great_expectations/datasource/fluent/schemas/RedshiftDatasource.json

```
@@ -1,6 +1,6 @@
 {
     "title": "RedshiftDatasource",
-    "description": "Adds a redshift datasource to the data context using psycopg2.\n\nArgs:\n    name: The name of this redshift datasource.\n    connection_string: The SQLAlchemy connection string used to connect to the redshift\n        database. This will use a redshift with psycopg2. For example:\n        \"redshift+psycopg2://username@host.amazonaws.com:5439/database\"\n    assets: An optional dictionary whose keys are TableAsset or QueryAsset names and whose\n        values are TableAsset or QueryAsset objects.",
+    "description": "--Public API--Adds a redshift datasource to the data context using psycopg2.\n\nArgs:\n    name: The name of this redshift datasource.\n    connection_string: The SQLAlchemy connection string used to connect to the redshift\n        database. This will use a redshift with psycopg2. For example:\n        \"redshift+psycopg2://username@host.amazonaws.com:5439/database\"\n    assets: An optional dictionary whose keys are TableAsset or QueryAsset names and whose\n        values are TableAsset or QueryAsset objects.",
```

This is maybe the wrong place to apply these changes (looks like this is generated json?) but here's a suggestion that does a few things:
- capitalizes Redshift as a proper noun
- removes what appears to be some stray words (when compared to the similar page for [postgres](https://deploy-preview-11097.docs.greatexpectations.io/docs/reference/api/datasource/fluent/postgresdatasource_class/)


```suggestion
    "description": "--Public API--Adds a Redshift datasource to the data context using psycopg2.\n\nArgs:\n    name: The name of this Redshift datasource.\n    connection_string: The SQLAlchemy connection string used to connect to the Redshift\n        database. For example:\n        \"redshift+psycopg2://username@host.amazonaws.com:5439/database\"\n    assets: An optional dictionary whose keys are TableAsset or QueryAsset names and whose\n        values are TableAsset or QueryAsset objects.",
```

---

I wanted to double check the connection string format here since it's different that what I'm using in my docs PR based on [feedback on my docs plan](https://docs.google.com/document/d/1w5iE34xj2A1FALUOOx9czgfmT8-_oLB4aH221RDUWng/edit?disco=AAABfmWhHa8)

This PR has 

`redshift+psycopg2://username@host.amazonaws.com:5439/database`

My PR has 

`redshift+psycopg2://<USER_NAME>:<PASSWORD>@<HOST>:<PORT>/<DATABASE>?sslmode=<SSLMODE>`

notable differences are:
- whether or not password is included
- whether or not SSL mode is included

Should the discrepancy between PRs be resolved? if yes, in what direction?

---

This should be resolved when @alena-hutchinson's PR merges. I will regenerate this after all other PRs to this file go in since this is the last PR to merge.

---

This has the updated changes now.